src/v2/views/FormView.js

```diff
@@ -36,10 +37,26 @@ define([
         }
       }).last();
 
-      this.listenTo(form, 'save', (model) => {
-        this.options.appState.trigger('saveForm', model);
-      });
+      this.listenTo(form, 'save', this.saveForm);
+
+      this.maybeRunPolling(remediationValue, model);
     },
 
+    maybeRunPolling (remediationValue, model) {
```

Comment:
Should we add a test for this one?

Reply:
test is hold on in general now but certainly will be added later.